tests/test_multi_server.py

```diff
@@ -8,6 +8,9 @@
 from ansys.dpf.core.server_factory import ServerConfig, CommunicationProtocols
 
 
+dpf.apply_server_context(dpf.AvailableServerContexts.entry)
```

Comment:
@cbellot000 Is this to test applying the entry context? Or for the "other_remote_server" fixture to return "entry" GrpcServers? Could that not be directly set in the fixture?